### ISSUE TITLE
Set versioning policy

### ISSUE THREAD

**Issue by delphinus** [nvim-telescope/telescope-frecency.nvim#278]
This project has no versioning such as git-tagged ones. I will introduce `0.9.0` tag for the master branch before merging #275, that is, dropping support for Neovim v0.9.x. When merged, it will be tagged `1.0.0` and Neovim v0.9.x users should be use `0.9.0` tagged commit.

If significant bugs are fixed in the future, we will backport them into `0.9.0` and release `0.9.1` or either.

## Recommended settings

```lua
{
  "nvim-telescope/telescope-frecency.nvim",
  -- install the latest stable version
  version = "*",
  config = function()
    require("telescope").load_extension "frecency"
  end,
}
```

or for Neovim 0.9.x users

```lua
{
  "nvim-telescope/telescope-frecency.nvim",
  -- install any compatible version of 0.9.x
  version = "^0.9.0",
  config = function()
    require("telescope").load_extension "frecency"
  end,
}
```

**Comment by delphinus** [comment]
https://github.com/nvim-telescope/telescope-frecency.nvim/releases/tag/0.9.0

I made 0.9.0. When #275 is merged, I will publish 1.0.0.

**Comment by delphinus** [comment]
I released 1.0.0. Close this.

https://github.com/nvim-telescope/telescope-frecency.nvim/releases/tag/1.0.0
